src/extension.ts

```diff
@@ -70,3 +75,16 @@ async function withSpinningStatus(context: vscode.ExtensionContext, action: (sta
 
 // this method is called when your extension is deactivated
 export function deactivate(): void {}
+
+class ExtensionAPI {
+
+    private kotlinApi?: KotlinAPI;
+
+    constructor(kotlinApi: KotlinAPI) {
+        this.kotlinApi = kotlinApi;
+    }
+
+    getBuildOutputPath(): string {
+        return this.kotlinApi?.getBuildOutputPath();
+    }
+}
```

Comment:
Trailing newlines would be great (the EditorConfig extension should automatically take care of this, otherwise it should be pretty easy to configure VSCode to do this)!